testing.rst

```diff
@@ -20,7 +20,7 @@ wraps the original PHPUnit binary to provide additional features:
 
 .. code-block:: terminal
 
-    $ composer require --dev symfony/phpunit-bridge
+    $ composer require --dev phpunit
```

Comment:
Not sure about this change. The `phpunit` thing is an alias to our `phpunit-bridge`... but if we use just `phpunit`, then the reader doesn't know if they are installing PHPUnit or "our PHPUnit". Also, I'm not sure about removing the `./vendor/bin/simple-phpunit` commands. What's the reasoning behind that?

Reply:
Thank you for you review, but I don't understand what you mean exactly by 
>if we use just phpunit, then the reader doesn't know if they are installing PHPUnit or "our PHPUnit"

are you talking of other contexts? Because here, we're saying that we need the bridge before showing the sample.

>about removing the ./vendor/bin/simple-phpunit commands

This is how it works with Flex now. We just need to run `./bin/phpunit` which is the bridge wrapper installed by the recipe, and is responsible for actually installing the phpunit's dependencies (ref https://github.com/symfony/recipes/blob/master/symfony/phpunit-bridge/3.3/bin/phpunit, and https://github.com/symfony/recipes/pull/277).

Reply:
We could add a note, maybe even a link, to say we use recipes here.

Reply:
Though I think I agree with Javier about not using the `phpunit` alias here

Reply:
Ok changed in https://github.com/symfony/symfony-docs/pull/9452/commits/9a6b058eb938a0c6e6bab64ceadc6f2d5c7b5471.
Is this ok for you? Should I remove the new note?